content/technotes/2019-03-18-drake-700.Rmd

```diff
@@ -0,0 +1,169 @@
+---
+slug: drake-700
+title: "drake transformed"
+date: 2019-03-18
+authors:
+  - Will Landau
+categories: technotes
+tags:
+  - R
+  - packages
+  - drake
+  - reproducibility
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+suppressPackageStartupMessages(library(drake))
+cache <- get_cache(search = FALSE)
+clean(cache = cache, destroy = TRUE)
+new_cache(path = ".drake")
+if (exists("mtcars", envir = globalenv(), inherits = FALSE)) {
+  rm(mtcars, envir = globalenv(), inherits = FALSE)
+}
+```
+
+Version 7.0.0 of [`drake`](https://github.com/ropensci/drake) just arrived on [CRAN](http://cran.r-project.org/), and it is faster and easier to use than previous releases.
+
+```{r install, eval = FALSE}
+install.packages("drake")
+```
+
+## Recap
+
+Data analysis can be slow. A round of scientific computation can take several minutes, hours, or even days to complete. After it finishes, if you update your code or data, your hard-earned results may no longer be valid. How much of that valuable output can you keep, and how much do you need to update? How much runtime must you endure all over again?
+
+For projects in R, the [`drake`](https://github.com/ropensci/drake) package can help. It [analyzes your workflow](https://ropenscilabs.github.io/drake-manual/plans.html), skips steps with up-to-date results, and orchestrates the rest with [optional distributed computing](https://ropenscilabs.github.io/drake-manual/hpc.html). At the end, [`drake`](https://github.com/ropensci/drake) provides evidence that your results match the underlying code and data, which increases your ability to trust your research.
```

Comment:
_(minor word smithing)_ For projects in R, the workflow package `drake` can help.